### ISSUE TITLE
Localized user-report SMS template

### ISSUE THREAD

**Issue by mikehelmick** [google/exposure-notifications-verification-server#2342]
### TL;DR

Currently, there is only one `User Report` template. Allow for this to be localized.

### Design

**Proposal**

For iOS (WebView), the desired language is known.
For Android (API), a new field will need to be added to the API.

For the verification server, a UI would need to be designed and built to allow for localizations of a specific SMS string (including setting the default / fallback template).

**Alternatives considered**

Don't do this / maintain status quo. So far we have not had a formal request for this.

**Resources**

Estimated LOE, 2 Days of SWE time do design, build, test.

**Additional information**

n/a

**Comment by mikehelmick** [comment]
No, I don't think this is necessary for 2 reasons.

1. Lack of user demand.
2. Many different SMS templates for a single source would not play well with SMS anti-abuse measures of wireless carriers.
